requirements.txt

```diff
@@ -1,7 +1,9 @@
-myst-parser~=0.18.0
-pydata-sphinx-theme==0.15.2
+sphinx~=8.0.0
+myst-parser~=4.0.0
```

Comment:
Crazy that we're jumping 4 major versions here :D I guess this wasn't updated in a while before.

Reply:
or python2.7 in aiida-plugin-cutter ;)